channels.c

```diff
@@ -3503,6 +3513,23 @@ channel_add_permitted_opens(char *host, int port)
 	all_opens_permitted = 0;
 }
 
+void
+channel_add_permitted_listens(char *host, int port)
+{
+	debug("allow remote port forwarding to host %s port %d", host, port);
+
+	permitted_listens = xreallocarray(permitted_listens,
+        num_permitted_listens + 1, sizeof(*permitted_listens));
```

Comment:
This indentation does not match [OpenBSD style guide](http://man.openbsd.org/style.9):

>  Indentation is an 8 character tab. Second level indents are four spaces. All code should fit in 80 columns.